openedx/core/djangoapps/xblock/rest_api/views.py

```diff
@@ -136,3 +136,16 @@ def xblock_handler(request, user_id, secure_token, usage_key_str, handler_name,
     # to call these handlers:
     response['Access-Control-Allow-Origin'] = '*'
```

Comment:
We should be able to remove this line, and the comments above, because the `cors_allow_xblock_handler` has the same effect (and also works for `OPTIONS` requests).

Reply:
That makes sense, removed.